docker-compose.yaml

```diff
@@ -1,43 +1,41 @@
-version: "3.8"
+version: '3.8'
 services:
-  db:
-    container_name: postgres
-    image: postgres
-    restart: always
-    healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U admin -d rps"]
-      interval: 3s
-      retries: 5
-      start_period: 30s
+  mongodb:
+    container_name: mymongo
+    image: mongo
+    restart: unless-stopped
     env_file:
-      - postgres.env
+      - mongo.env
     volumes:
-      - postgres-data:/var/lib/postgresql/data
-    ports:
-      - 5432:5432
+      - ./.docker/mongodb/data/db/:/data/db/
```

Comment:
Le'ts use *named* volumes instead of mapping to a folder

e.g.
`mongo-data:/data/db/`
`mongo-logs:/var/log/mongodb/`

and define these named volumes at the bottom of the yaml file